### PR TITLE
Add timezone field to Organizations and exposed allowed_payment_processors to API endpoints

### PR DESCRIPTION
**Pull request by philipr-za**
### Closes Issues:
Closes https://github.com/big-neon/bn-api/issues/986
Related to: https://github.com/big-neon/bn-api/pull/984 and https://github.com/big-neon/bn-api/issues/937
### Description:
Added the timezone field to Orgs and exposed the allowed_payment_processors field to the API create and update endpoints. 

### Migrations
20190206111804_add_timezone_and_new_default_payment_provider_to_organizations

### Environment Variables
 * No change

### CODE REVIEW

File: db/migrations/20190206111804_add_timezone_and_new_default_payment_provider_to_organizations/up.sql [hunk 1]
```diff
@@ -0,0 +1,8 @@
+alter table organizations
```
Comment (Krakaw) [inline db/migrations/20190206111804_add_timezone_and_new_default_payment_provider_to_organizations/up.sql]: A really minor thing, the styling convention that most of the migrations use is:
```sql
ALTER TABLE organizations
  ADD timezone TEXT,
  ALTER allowed_payment_providers SET DEFAULT '{"Stripe"}';
```

File: db/migrations/20190206111804_add_timezone_and_new_default_payment_provider_to_organizations/down.sql [hunk 2]
```diff
@@ -0,0 +1,8 @@
+alter table organizations
```
Comment (Krakaw) [inline db/migrations/20190206111804_add_timezone_and_new_default_payment_provider_to_organizations/down.sql]: Could be bundled:
```sql
ALTER TABLE organizations
  DROP COLUMN timezone,
  ALTER allowed_payment_providers SET DEFAULT '{"stripe"}';
```

File: db/migrations/20190206111804_add_timezone_and_new_default_payment_provider_to_organizations/up.sql (hunk 1 above)
Comment (philipr-za) [reply db/migrations/20190206111804_add_timezone_and_new_default_payment_provider_to_organizations/up.sql]: done

File: db/migrations/20190206111804_add_timezone_and_new_default_payment_provider_to_organizations/down.sql (hunk 2 above)
Comment (philipr-za) [reply db/migrations/20190206111804_add_timezone_and_new_default_payment_provider_to_organizations/down.sql]: Done

**Pull request merged**